README.md

```diff
@@ -2,6 +2,8 @@
 
 This web application is developed by Mojang's Mojira Team, the bug tracker helpers and moderators. "Mojira Helper Message Application" aggregates & serves copy + paste messages for use in comments on the bug tracker.
 
+⚠ Please note that the helper messages may only be used on the bug tracker by mods, helpers, or Mojang staff. Do not use them yourself if you do not belong to one of these groups as it may confuse users regarding your role.
```

Comment:
```suggestion
⚠ Please note that the helper messages may only be used on the bug tracker by helpers, moderators or Mojang staff. Do not use them yourself if you do not belong to one of these groups as it may confuse users regarding your role.
```
Write out "moderator" since this is how it is written further down below as well.